crates/portalnet/src/types/kbucket.rs

```diff
@@ -754,6 +759,50 @@ mod tests {
         }
     }
 
+    mod random {
+        use super::*;
+
+        #[test]
+        fn simple() {
+            let (_local_enr, kbuckets) = create_kbuckets_table();
+            let content_id = NodeId::random().raw();
+
+            let nodes_per_distance = 5;
+
+            // Distance 254 and disconnected
+            let enr_dis_254_disconnected = (0..nodes_per_distance)
```

Comment:
```suggestion
            let enr_dis_254_disconnected = (0..nodes_per_distance)
```

I wish `dis` -> was `distance`, it isn't a big deal because these are tests, but it would be nice :grin: